test/graphql_generator_test.dart

```diff
@@ -42,4 +42,57 @@ void main() {
       resultFile.delete();
     }
   });
+
+test('GraphQL generation', () async {
+    /// Read the test response from the file.
+    var graphQLGenerator = GraphQLGenerators();
+    var response = await graphQLGenerator.getSchema("https://turtle-api.herokuapp.com/graphiql", "", introspectionQuery:
```

Comment:
Let us avoid calling live APIs. What if the schema changes? See my comment in Turtle about generating your own test introspection schema.

Reply:
@venkatd If we are not using a live API we would have to pass the response JSON directly to the library. Is that how we need to do this?